### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by zliu41**
Removed "security report" and made all fields optional.

### CODE REVIEW

**Review by Unisay — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/feature_request.yml
Comment (effectfully) [inline .github/ISSUE_TEMPLATE/feature_request.yml:1]: Remove the `enhancement` label that this template adds?
Also, we discussed during the call that this probably should be more general than just a feature request. Something more general like "tell us what you think" or whatever.

Comment (zliu41) [reply .github/ISSUE_TEMPLATE/feature_request.yml:1]: > Something more general like "tell us what you think" or whatever.

I think people can create a blank issue for that.

Comment (effectfully) [reply .github/ISSUE_TEMPLATE/feature_request.yml:1]: It was rather non-obvious to me that the blank issue template existed. I know it's right on that page, but it's smaller than the templates, looks differently and there are links all over the place, so I did miss it a number of times before finally finding it hiding in plain sight. Dunno, maybe it's just me.

Comment (zliu41) [reply .github/ISSUE_TEMPLATE/feature_request.yml:1]: Yeah but I don't see a way to remove the blank issue option, and having two options for blank issues is more confusing IMO

Comment (effectfully) [reply .github/ISSUE_TEMPLATE/feature_request.yml:1]: Good point.

**Pull request merged**